.cirrus.yml

```diff
@@ -50,12 +50,19 @@ release_task:
   upload_script: |
     #!/usr/bin/env bash
 
-    if [[ "$CIRRUS_RELEASE" == "" ]]; then
+    NUM_TRIES=0
+    until [ $NUM_TRIES -eq 5 ]
```

Comment:
Considering in the best case scenarios cirrus takes around 4 minutes to get here and the release workflow in github actions takes on average around 8 minutes to get to "Create release" step, I would say we should probably wait at the very least for 4 minutes. That said, I am more inclined to suggest a 10 minute hard limit at 30 second intervals. So 20 retries. The current retries might be insufficient for what we need.

Reply:
@abn thank you for the feedback. i updated the PR here: https://github.com/python-poetry/poetry/pull/3181/commits/a0847594097b2a3be6aa3babfeaa049be15a179f. wdyt?

Reply:
One way to test it is to create a cirrus job on your own fork and link the test runs. However, I am okay to waive the requirement for this one and we will just test it at next release. :)

Reply:
Just let me know if you would want to try make a test out of interest.